### ISSUE TITLE
Autocomplete accessibility glitch

### ISSUE THREAD

**Issue by ChaseKnowlden** [ChaseKnowlden/vscode#5]
From vscode created by [zersiax](https://github.com/zersiax): microsoft/vscode#3787  

There's something odd going on when typing actual code into Vs Code. To reproduce:
1. Create a file called index.html.
2. Type <html> .
3. Press enter.
Expected behavior:
The cursor goes to the next line.
Perceived behavior:
Nothing actually happens. You have to press Enter again to go to the next line.

Expected cause: I think an autocomplete list pops up that the accessibility stack isn't aware of. Pressing the first enter likely selects the first option in this list.
Problem with this hypothesis: Arrowing up and down should make different things happen if my theory is correct, however it doesn't seem to do that. Therefore I am not a 100% sure I am correct, someone comment on it to check please.
I think fixing this so the prsumed opup is correctly handled should be the next step for accessibility of this product.
